packages/rich-text/src/create.js

```diff
@@ -419,6 +424,24 @@ function createFromElement( {
 			attributes: getAttributes( { element: node } ),
 		} );
 
+		if ( format?.formatType?.contentEditable === false ) {
+			delete format.formatType;
+			accumulateSelection( accumulator, node, range, createEmptyValue() );
+			mergePair( accumulator, {
+				formats: [ , ],
+				replacements: [
+					{
+						...format,
+						innerHTML: node.innerHTML,
+					},
+				],
+				text: OBJECT_REPLACEMENT_CHARACTER,
+			} );
+			continue;
+		}
+
+		if ( format ) delete format.formatType;
```

Comment:
What are these changes about? Do we need to test them?

Reply:
Added a comment and a unit test